R/kobo.R

```diff
@@ -49,12 +41,16 @@ Kobo <- R6::R6Class("Kobo",
             if (version == "v2") {
```

Comment:
The optional argument in query is not really helpful: if what we mean is that by default the get function should request output in JSON format, I would include it as an extra argument to the function. In fact, if a user writes a query list now, the optional format will be overridden. 
I would propose 
`function(path, query= None, version = "v2", format="json") `
and then always append to the query a format (in line 42 and 48, or before )

Reply:
Sorry, I could not comment on the line before...